FALSEHOODS.md

```diff
@@ -0,0 +1,79 @@
+# Falsehoods Programmers Believe About Phone Numbers
+
+... and tips on how to use libphonenumber.
+
+Given how ubiquitous phone numbers are and how long they've been around, it's surprising how many false assumptions programmers continue to make about them.
+
+1.  **Phone numbers that are valid today will always be valid. Phone numbers of a certain type today (e.g., mobile) will never be reassigned to another type.**
+
+    A phone number which connects today may be disconnected tomorrow. A number which is free to call today may cost money to call tomorrow. The phone company may decide to expand the range of available phone numbers by inserting a digit into an existing number.
```

Comment:
Sure. But the README.md and CONTRIBUTING.md are not wrapped at 80, so it's inconsistent either way.